public/javascripts/SVLabel/src/SVLabel/canvas/ContextMenu.js

```diff
@@ -249,6 +259,15 @@ function ContextMenu (uiContextMenu) {
         });
     }
 
+    function removeLabelAndUpdateUI(labelName, labelTags) {
```

Comment:
I would suggest having this function return the modified labelTags parameters instead of just silently modifying it.